src/components/plots/rectanglePlot.ts

```diff
@@ -21,8 +21,17 @@ export module Plots {
      */
     constructor(xScale: Scale<X, any>, yScale: Scale<Y, any>) {
       super(xScale, yScale);
+
+      this.animator("cells", new Animators.Null());
```

Comment:
Up to you since we probably want to refine this endpoint anyways, but I think the string should instead be "rectangles"